### DISCUSSION TITLE
分享买房避坑经验，降低买到烂尾或者问题房的概率

### DISCUSSION THREAD

**Discussion by AviorAlong** [WeNeedHome/SummaryOfLoanSuspension#295] [General]
本来以为，正规渠道买大房企就不会有大问题，现在看来，没多大用，希望有经验的大佬来这分享下避坑指南

**Comment by jingzz1** [comment]
买现房！

**Comment by kiss5523** [comment]
不要期房

**Comment by superceII** [comment]
买负债率低的大房企，比如：招商、保利、万科、金地、华润

**Comment by qq912276337** [comment]
现在这个阶段还想着买房？？？是跪久了是吧 不怕失业？？？

**Comment by DoDayum** [comment]
就是别买河南的房子，有点太恐怖了

**Comment by haoliangwu** [comment]
不买期房就行了，这东西本来就是坑人的，贪小便宜吃大亏，另外这个时间点的话，对于普通人，不建议通过任何形式加杠杆，不论是买房、买车还是炒 xxx 东西。

**Comment by jiang443** [comment]
很简单，见不到房子不签字、不给钱。（样板房不算数）
